scss/_navbar-offcanvas.scss

```diff
@@ -156,7 +158,7 @@
 
     .dropdown-item {
       padding: $navbar-offcanvas-dropdown-link-padding;
-      font-weight: 200;
+      font-weight: 400;
```

Comment:
```suggestion
```
Recommend removing since it's being set to the same value as the default .dropdown-item class.